lib/classes/compatibility/buddyboss.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Plugin Name: BuddyPress
```

Comment:
Is it correct? Not BuddyBoss?

Reply:
It should have been `BuddyBoss`. Updated the text.